connection.go

```diff
@@ -569,21 +605,23 @@ func (v *connection) initializeSSL(sslFlag string) error {
 	}
 
 	switch sslFlag {
-	case "server":
+	case tlsModeServer:
 		connectionLogger.Info("enabling SSL/TLS server mode")
 		v.conn = tls.Client(v.conn, &tls.Config{InsecureSkipVerify: true})
-	case "server-strict":
+	case tlsModeServerStrict:
 		connectionLogger.Info("enabling SSL/TLS server strict mode")
 		v.conn = tls.Client(v.conn, &tls.Config{ServerName: v.connURL.Hostname()})
 	default:
-		err := fmt.Errorf("unsupported tlsmode flag: %s - should be 'server', 'server-strict' or 'none'", sslFlag)
-		connectionLogger.Error(err.Error())
-		return err
+		connectionLogger.Info("enabling SSL/TLS custom mode")
+		config, ok := tlsConfigs.get(sslFlag)
+		if !ok {
+			err := fmt.Errorf("tls config %s not registered", sslFlag)
```

Comment:
I think it is better to provide guidance about how to register with "RegisterTLSConfig()" in this error message.

Reply:
done (referred to the README.md which has an example on how to register and use a custom config)